src/script/lua_api/l_env.cpp

```diff
@@ -651,38 +651,49 @@ int ModApiEnvMod::l_find_nodes_in_area(lua_State *L)
 	INodeDefManager *ndef = getServer(L)->ndef();
 	v3s16 minp = read_v3s16(L, 1);
 	v3s16 maxp = read_v3s16(L, 2);
```

Comment:
please prevent underflow in the next calculation (ordering max and min or verify it's correctly ordered)